src/fdtdx_studio/ui/panels/volume_panel.py

```
@@ -31,36 +37,39 @@ def Volume_panel(self):
-      with ui.row():
-        ui.button("Small", on_click= lambda: preset(5e-6,5e-6,5e-6)).tooltip('Sets size to 5µm x 5µm x 5µm')
-        ui.button("Medium",on_click= lambda: preset(1e-5,1e-5,1e-5)).tooltip('Sets size to 10µm x 10µm x 10µm')
-        ui.button("Large", on_click= lambda: preset(1e-4,1e-4,1e-4)).tooltip('Sets size to 100µm x 100µm x 100µm')
+    def preset(W,H,L):
+      nonlocal x,y,z
+      x.value = W
+      y.value = H
+      z.value = L
+      
+    def preset_changed(val):
+      if val == 'Small': preset(5e-6, 5e-6, 5e-6)
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)